bindings/matrix-sdk-ffi/src/room_info.rs

```diff
@@ -54,8 +56,10 @@ pub struct RoomInfo {
     /// Events causing mentions/highlights for the user, according to their
     /// notification settings.
     num_unread_mentions: u64,
-    /// The currently pinned event ids
+    /// The currently pinned event ids.
     pinned_event_ids: Vec<String>,
+    /// The join rule for this room, if known.
+    join_rule: Option<JoinRule>,
```

Comment:
should we remove the `is_public` as it's checking the `JoinRule`?

Reply:
I don't see a `is_public` field in `RoomPreviewInfo`.

Reply:
Yep, good call.